templates/taxi/car_confirm_delete.html

```diff
@@ -0,0 +1,12 @@
+{% extends "base.html" %}
+
+{% block content %}
+    <h1>Delete Car</h1>
+
+    <p>Are you sure you want to delete the car: {{ car }}</p>
+
+    <form action="" method="POST">
+        {% csrf_token %}
+        <input class="btn btn-danger" type="submit" value="Confirm"/>
```

Comment:
Add Cancel button apart from Delete one. The Cancel button will lead to the previous page the user was on -- checklist